vnext/local-cli/runWindows/utils/deploy.js

```diff
@@ -159,7 +159,13 @@ async function deployToDevice(options, verbose) {
   }
 }
 
-async function deployToDesktop(options, verbose) {
+function hasDotNetProjects(slnFile) {
+  const contents = fs.readFileSync(slnFile).toString();
```

Comment:
We shouldn't make this block. Use `fs.promises.readFile` and make this an async function.

Reply:
what's the point of that? we are going to block on reading and matching it anyway

Reply:
It blocks any background work we might want to do, like showing a spinner, doing other reads, etc.